kitsune/wiki/templates/wiki/mobile/document.html

```diff
@@ -31,8 +31,13 @@
 {% endif %}
 
 {% block content %}
-  <div class="title-bar">{{ document_title(document) }}</div>
+  {% if breadcrumb_items[1] %}
+    <div class="filter-bar">
+      <a class="return-link" href="{{ breadcrumb_items[1][0] }}">{{ breadcrumb_items[1][1] }}</a>
```

Comment:
Will this work with documents in sub-topics? I'm guessing that `breadcrumb_items[1]` will always be the top level topic.